### ISSUE TITLE
Filled input fields 

### ISSUE THREAD

**Issue by ViRuSTriNiTy** [mdbootstrap/mdb-ui-kit#1427]
Hi there,

I have noticed that your framework supports the outlined type for input fields only, but Material Design also specifies the type `Filled`, see https://material.io/components/text-fields.

I currently use your framework in connection with another framework which uses the filled type.

Now, I would like to switch to the filled type in your framework but I somewhat hesitate to change all the styles to accomplish this as you might have something in the works.

TLDR: Are you going to support the filled type for input fields?

So lonG
Daniel

**Comment by ViRuSTriNiTy** [comment]
Ok, I have fiddled something together that matches the filled type pretty closely by:

SCSS variables override
```
// shift the input label a bit more in the upper direction to ensure label does not overlay the input text
$input-focus-active-label-transform: translateY(-1.25rem /* original value -1rem */) translateY(0.1rem) scale(0.8) ;
```

Custom CSS
```
.form-outline .form-control {
    padding-left: 0;
	margin-top: 0.5rem;
}

.form-outline .form-control ~ .form-label {
    left: 0;
}

.form-outline .form-control ~ .form-notch .form-notch-leading {
    display: none;
}

.form-outline .form-control ~ .form-notch .form-notch-middle {
    border-top: 0;
}

.form-outline .form-control ~ .form-notch .form-notch-trailing,
.form-outline .form-control:focus ~ .form-notch .form-notch-trailing {
    border-top: 0;
    border-right: 0;
    border-radius: 0;
}
```
Applied to your doc pages it looks like this:

![image](https://user-images.githubusercontent.com/12170676/105217060-ad218380-5b53-11eb-8062-45fbcaaf2547.png)

This could be used as a starter template for other users when the filled type will not be supported officially.

**Comment by smolenski-mikolaj** [comment]
Thanks for sharing your solution. We decided to support only one type of inputs to ensure consistency in all examples.
